docs/source/docs/quick-start/camera-calibration.md

```diff
@@ -31,3 +31,293 @@ If you’re not using cameras in 3D mode, calibration is optional, but it can st
   - A couple of up close images is good.
   - Cover the entire cameras fov.
   - Avoid images with the board facing straight towards the camera.
+
+## Interactive Camera Transformation
+
+Below is an interactive demo to visualize multiple cameras' positions and orientations relative to the robot chassis. Use the table to adjust each camera's transformation parameters (position and rotation).
+
+<div id="camera-demo" style="width: 800px; height: 400px; border: 1px solid #ccc; margin: auto;"></div>
+<table id="camera-table" style="table-layout: fixed; width: 100%;">
+  <thead>
+    <tr style="height: 35px;">
+      <th>Camera</th>
+      <th>Position X</th>
+      <th>Position Y</th>
+      <th>Position Z</th>
+      <th>Roll (deg)</th>
+      <th>Pitch (deg)</th>
+      <th>Yaw (deg)</th>
+      <th>Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+  </tbody>
+</table>
+<button id="add-camera" style="margin-top: 10px;">Add Camera</button>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r134/three.min.js"></script>
```

Comment:
This will not work offline. We should figure out how to make this work offline lol